stacker_blueprints/efs.py

```diff
@@ -0,0 +1,123 @@
+from troposphere import ec2, efs
+from troposphere import Join, Output, Ref, Split
+
+from stacker.blueprints.base import Blueprint
+from stacker.blueprints.variables.types import EC2VPCId, EC2SecurityGroupIdList
+
+
+class EFSBlueprint(Blueprint):
+    VARIABLES = {
+        'VpcId': {
+            'type': EC2VPCId,
+            'description': 'VPC ID to create resources'
+        },
+        'PerformanceMode': {
+            'type': str,
+            'description': 'The performance mode of the file system',
+            'default': 'generalPurpose'
+        },
+        'FileSystemTags': {
+            'type': dict,
+            'description': 'Tags to associate with the file system.',
+            'default': {}
+        },
+        'Subnets': {
+            'type': str,
+            'description': 'Comma-delimited list of subnets to deploy private '
+                           'mount targets in.'
+        },
+        'SecurityGroups': {
+            'type': str,
+            'description': 'Security groups to place mount targets in. '
+                           'Omit to create automatically from '
+                           'AllowedCIDRs',
+            'default': ''
+        },
+        'IPAddresses': {
+            'type': str,
+            'description': 'List of IP addresses to assign to mount targets. '
+                           'Omit to assign automatically. '
+                           'Corresponds to Subnets listed in the same order.',
+            'default': ''
+        },
+        'AllowedCIDRs': {
```

Comment:
Very fair point, I'll change it to use a TroposphereType entry. I actually gave up on EFS entirely though - it was unacceptably slow to store a small amount of data for website hosting :(